src/js/types.ts

```diff
@@ -261,6 +261,7 @@ export interface FSType {
   syncfs(dir: boolean, oncomplete: (val: void) => void): void;
   findObject(a: string, dontResolveLastLink?: boolean): any;
   readFile(a: string): Uint8Array;
+  sitePackages: string;
```

Comment:
Not sure about this. Do we really want to extend `FS` object? If we want to make it public to users, adding it directly to the API would be better for documentation and visibility.

Reply:
Well I want to give the `fsInit` function access to it but the api itself isn't ready yet at that point.

Reply:
The other option I considered was just passing it as an argument. I do think we should expose it somewhere.

Reply:
> The other option I considered was just passing it as an argument.

This sounds better to me. I would like to keep the object that comes from the Emscripten untouched as much as possible (Module is kind of an exception though)

Reply:
Okay I will switch it.

Reply:
We could avoid changing `Module` a lot more than we do by favoring putting stuff on `API` or other objects we make.

Reply:
Sounds good. One issue is that functions that come from the native code become attributes of the `Module`, which inevitably leads to a lot of cases where we need to access the Module. I guess we can distinguish the _Emscripten_ Module and _Pyodide_ Modules in a typing perspective, if we adopt DefinitelyTyped.